md/performance-tuning.md

```diff
@@ -170,6 +170,23 @@ Setting a high `queueCapacity` limit means that more work can be queued, but can
 It is essential to ensure that the queue never becomes full (`queueCapacity` is never reached).  
 If the queue becomes full, the application restarts in order to force the engine to generate all work from the database. This means that work is lost.
 
+##### SQLServer
+
+Sometimes, in case of a lot of concurrence ( work triggered in same ms ), the data commits in a database was not visible by the next transaction.
+
+It happens when the previous transaction has used  **XAMultipleResource (Bonita + BDM )** and when the **transaction isolation level** is configured  **ALLOW_SNAPSHOT_ISOLATION** and **READ_COMMITTED_SNAPSHOT** are configured. Those isolations level are mandatory to avoid a deadlock.
```

Comment:
```suggestion
This issue happens only when using Bonita and BDM XA resources ( **XAMultipleResource** ) and because the **transaction isolation level** is configured as **ALLOW_SNAPSHOT_ISOLATION** and **READ_COMMITTED_SNAPSHOT**. These isolation levels are mandatory to avoid a deadlock.
```